messaging/ses/src/sendEmail.ts

```diff
@@ -65,15 +61,19 @@ export default async (event: FunctionEvent<EventData>) => {
       Source: from
     }
 
-    ses.sendEmail(params, (err, data) => {
-      if (err) {
-        console.log(err, err.stack)
-      } else {
-        response => response.json()
-      }
+    ses.sendEmail(params, (err, response) => {
+      return new Promise((resolve, reject) => {
+        if (err) {
```

Comment:
you got the promise wrapping backwards - check the example I shared in the previous review 🙂

Reply:
If I wrap it the normal way, I can't `return { data: { success: true } }`

Reply:
Actually, I'll just submit an issue because when I do it the right way, it's successful but the console throws me an error

Reply:
From within the promise you should not `return`, but `resolve({data: { sucess: true})`

Reply:
Nevermind. I figured it out. Haha

Reply:
You wouldn't be able to do that

Reply:
I tried that but it didn't actually work.